src/fragments/lib/datastore/native_common/sync-distributed-data.mdx

```diff
@@ -20,6 +20,10 @@ import android2 from "/src/fragments/lib/datastore/android/sync/20-savePredicate
 
 <Fragments fragments={{android: android2}} />
 
+import flutterSavePredicateExample from "/src/fragments/lib/datastore/flutter/sync/20-savePredicate.mdx";
+
+<Fragments fragments={{flutter: flutterSavePredicateExample}} />
+
 There's a difference between the traditional local condition check using `if/else` constructs and the predicate in the aforementioned APIs as you can see in the example below.
```

Comment:
Maybe change aforementioned and just say `save()` and `delete()` APIs. Just to simplify the language a bit.

Reply:
Sounds good